pkg/gameservers/controller_test.go

```diff
@@ -1452,30 +1458,43 @@ func TestControllerGameServerPod(t *testing.T) {
 	}
 
 	t.Run("no pod exists", func(t *testing.T) {
-		c, gs, _, stop, cancel := setup()
+		c, gs, _, _, cancel := setup()
```

Comment:
It looks like stop is never used from `setup()` now so it can just be removed as a return var.

Reply:
Just found where we do use this (and shows an example of what you mentioned below)
https://github.com/markmandel/agones/blob/66ce69bf64c1e58b0e38fd642556e282ee70d060/pkg/gameservers/controller_test.go#L1504-L1512